cmd/quadlet/main.go

```diff
@@ -9,8 +9,8 @@ import (
 	"path/filepath"
 	"strings"
 
-	"github.com/containers/podman/v4/pkg/quadlet"
-	"github.com/containers/podman/v4/pkg/systemdparser"
+	systemdparser "github.com/containers/podman/v4/pkg/systemd/parser"
```

Comment:
Can we not rename this? I do not really like import renames unless required to name conflict.

Reply:
`"github.com/containers/podman/v4/pkg/systemd/parser"` is fine for me, just do not rename the package here.